src/SOS/SOS.UnitTests/Scripts/StackAndOtherTests.script

```diff
@@ -72,9 +72,12 @@ VERIFY:\s+<HEXVAL>\s+<HEXVAL>\s+SymbolTestApp\.Program\.Foo1\(.*\)\s+\[(?i:.*[\\
 VERIFY:\s+<HEXVAL>\s+<HEXVAL>\s+SymbolTestApp\.Program\.Main\(.*\)\s+\[(?i:.*[\\|/]SymbolTestApp\.cs) @ 19\]\s*
 SOSCOMMAND:SetSymbolServer -disable
 
+# Issue: https://github.com/dotnet/diagnostics/issues/1567
```

Comment:
Maybe revert changes to this as you added this in a separate PR

Reply:
I'll rebase and they should go away.